src/main/java/org/isf/medicalstockward/gui/WardPharmacyRectify.java

```diff
@@ -528,28 +529,24 @@ private MedicalWard chooseLot(Medical medical, boolean newLot) {
 		}
 		return medWard;
 	}
-	private boolean isAutomaticLot() {
-		return GeneralData.AUTOMATICLOT_IN;
-	}
 
-	protected Lot askLot() {
+	protected Lot askLot(Medical med) {
+		LocalDateTime preparationDate;
+		LocalDateTime expiringDate;
 		Lot lot = null;
-		JTextField lotNameTextField = new JTextField(15);
 
+		JTextField lotNameTextField = new JTextField(15);
+		lotNameTextField.addAncestorListener(new RequestFocusListener());
 		TextPrompt suggestion = new TextPrompt(MessageBundle.getMessage("angal.medicalstock.multiplecharging.lotid"), lotNameTextField);
-		suggestion.setFont(FONT_PLAIN_14);
+		suggestion.setFont(new Font("Tahoma", Font.PLAIN, 14)); //$NON-NLS-1$
```

Comment:
Curious as to why you only changed this CONSTANT to  its expansion and then didn't remove the definition at line 88?

Reply:
just applying most recent code from MovStockMultipleCharging, apparently this one was more optimized... I will rollback this. Thanks!